SIRO_README.md

```diff
@@ -9,9 +9,11 @@ Project-specific README for SIRo.
 1. Install Habitat-sim `main` branch.
     * [Build from source](https://github.com/facebookresearch/habitat-sim/blob/main/BUILD_FROM_SOURCE.md), or install the [conda nightly build](https://github.com/facebookresearch/habitat-sim#recommended-conda-packages).
         * Be sure to include Bullet physics, e.g. `python setup.py install --bullet`.
+        * Be sure not install `stable` branch of Habitat-sim since some functionality is not supported
     * Anecdotally, building from source is working more reliably (versus the conda nightly build).
     * If you build from source, configure `PYTHONPATH` and ensure that Python `import habitat_sim` imports your locally-built version of Habitat-sim.
     * Keep an eye on relevant [commits to main](https://github.com/facebookresearch/habitat-sim/commits/main) to help you decide when to update/rebuild Habitat-sim.
+    * There is an issue about cmake when building from source using Mac `missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun`. You can solve this by first install the tools `xcode-select --install`.
```

Comment:
This is very machine specific. It seems to happen only on your machine (at least it is not happening on my machine). I'm not sure it has to be specified in this readme.

Reply:
I think it would be nice to add this. If someone else counter the same problem again, then they have know this immediately.

Reply:
My point is that this issue is habitat-sim installation specific, not SIRo specific. That's why I'm not sure if it needs to be in this readme. But this is not critical.